admin/server/templates/index.jade

```diff
@@ -65,18 +65,20 @@ html
 		if wysiwygOptions
 			script.
 				Keystone.wysiwyg = { options: !{JSON.stringify(wysiwygOptions)} };
-
+				
+		// Initialize Google Analytics
+		script
+			| window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
```

Comment:
Need to get the `ga` variable in before any code in our bundle can call it, which is why this script tag is above the bundles